bpf/bpf_lxc.c

```diff
@@ -473,7 +473,7 @@ int tail_handle_ipv6(struct __ctx_buff *ctx)
 	struct ipv6hdr *ip6;
 	int ret;
 
-	if (!revalidate_data(ctx, &data, &data_end, &ip6))
+	if (!revalidate_data_pull(ctx, &data, &data_end, &ip6))
```

Comment:
Would bpf_skb_pull_data error out if the skb is linear?

Reply:
I don't think so. It should be no op if it's linear already. The same revalidate_data_pull is already used in bpf_host and bpf_overlay